client/termdb/TermdbVocab.js

```diff
@@ -991,15 +991,13 @@ export class TermdbVocab extends Vocab {
 		//TODO use the PresetNumericBins type for defaultBins
 		const defaultBins = await this.getDefaultBins({ tw })
 		if ('error' in defaultBins) throw defaultBins.error
-
-		const termIsFrozen = Object.isFrozen(tw.term)
-		if (termIsFrozen) tw.term = structuredClone(tw.term)
-
```

Comment:
@congyu-lu Please note the changes and comments here, the previous code was not allowing term.bins to be set on the original `term` copy.